storage/src/adb/any/variable/mod.rs

```diff
@@ -341,7 +376,6 @@ impl<E: RStorage + Clock + Metrics, K: Array, V: Codec, H: CHasher, T: Translato
                 .rewind_to_offset(prune_to_section, end_offset)
                 .await?;
             self.log.sync(prune_to_section).await?;
-            self.log_size = end_loc;
```

Comment:
This is already set in the loop above

Reply:
Ah, I guess you need to set it inside the loop since current_index is updated instead of where log_size used to be updated.